src/avram/errors.cr

```diff
@@ -93,4 +93,55 @@ module Avram
       super error
     end
   end
+
+  class PGClientNotInstalledError < AvramError
+    def initialize(original_message : String)
+      error = String.build do |message|
+        message << <<-ERROR
+        Message from Postgres:
+
+          #{original_message}
+  
+        Try this...
+  
+          ▸ If you are on macOS  you can install postgres tools from #{macos_postgres_tools_link}
+          ▸ If you are on linux you can try running #{linux_postgres_installation_instructions}
+          ▸ If you are on CI or some servers, there may already be a database created so you don't need this command"
+  
+  
+        ERROR
+      end
```

Comment:
Not that it's a problem, but I'm curious about why the use of `String.build` instead of passing the heredoc to `super`.

Reply:
Because I wrote this while I was still waking up 😛  lol. It's unnecessary for sure 😂